### PR TITLE
Adding format flag for json, yaml, or env

### PR DESCRIPTION
**Pull request by joshfinnie**
This PR will add a `--yaml` flag for `gordon get` to export the parameters in a yaml format instead of json format.

### CODE REVIEW

**Comment by cmac1000** [comment]
design-wise, maybe a `--format` option, defaulting to json, would be better than a `-yaml` boolean: we might want, for example, to support `.env`-style output too.

**Comment by joshfinnie** [comment]
that actually makes a lot of sense... Let me see what I can do.

**Comment by cmac1000** [comment]
I don't think the env format is really documented anywhere I know of, but all the examples I've seen don't have commas: https://gist.github.com/ericelliott/4152984

**Comment by cmac1000** [comment]
long-term, we should invert the code style and use dependency injection to make end-to-end testing of this kind of thing easier - now that we're adding new output formats, etc, our testing structure is showing its flaws.

Doesn't block this PR, though.

File: cmd/get.go [hunk 1]
```diff
@@ -2,18 +2,42 @@ package cmd
...
 	"github.com/pbs/gorson/internal/gorson/io"
 	"github.com/pbs/gorson/internal/gorson/json"
 	"github.com/pbs/gorson/internal/gorson/util"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
 )
 
+var format string
+
 func get(path string) {
```
Comment (joshfinnie) [inline cmd/get.go]: get can be a switch here too... any thoughts on one or the other?

```
func get(path string) {
        p := util.NewParameterStorePath(path)
        pms := io.ReadFromParameterStore(*p)
        switch format {
        case "yaml", "yml":
                serialized, err := yaml.Marshal(pms)
                if err != nil {
                        log.Fatal(err)
                }
                fmt.Println(string(serialized))
        case "env":
                serialized, err := yaml.Marshal(pms)
                if err != nil {
                        log.Fatal(err)
                }
                var re = regexp.MustCompile(`(\w*):\s`)
                fmt.Println(re.ReplaceAllString(string(serialized), "${1}="))
        case "json":
                marshalled := json.Marshal(pms)
                fmt.Println(marshalled)
        default:
                log.Fatal("No proper format requested. (yaml, env, json allowed)")
        }
}
```

Comment (cmac1000) [reply cmd/get.go]: I think case/switch is more idiomatic in go, but this is fine. No strong opinion :)

**Pull request merged**